File: app/src/main/java/com/uncmorfi/balance/model/UsersContract.java
package com.uncmorfi.balance.model;

import android.provider.BaseColumns;

public class UsersContract {
    public static abstract class UserEntry implements BaseColumns {
        static final String TABLE_NAME = "user";

        public static final String CARD = "card";
        public static final String NAME = "name";
        public static final String TYPE = "type";
        public static final String IMAGE = "image";
        public static final String BALANCE = "balance";
        public static final String EXPIRATION = "expiration";
        public static final String LAST_UPDATE = "last_update";
    }
}

File: app/src/main/java/com/uncmorfi/models/UsersContract.kt
package com.uncmorfi.models

import android.provider.BaseColumns

class UsersContract {
    abstract class UserEntry : BaseColumns {
        companion object {
            internal const val TABLE_NAME = "user"

            const val ID = BaseColumns._ID
            const val CARD = "card"
            const val NAME = "name"
            const val TYPE = "type"
            const val IMAGE = "image"
            const val BALANCE = "balance"
            const val EXPIRATION = "expiration"
            const val LAST_UPDATE = "last_update"
        }
    }
}
